client/src/components/Form/Elements/FormDialog.vue

```diff
@@ -0,0 +1,68 @@
+<script setup lang="ts">
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faFolderOpen } from "@fortawesome/free-solid-svg-icons";
+import DataDialog from "../../DataDialog/DataDialog.vue";
+import { onMounted } from "vue";
+import { getCurrentGalaxyHistory } from "../../../utils/data";
+import { reactive } from "vue";
+
+library.add(faFolderOpen);
+
+interface DataDialogProps {
+    id: string;
+    multiple: boolean;
```

Comment:
Since multiple is an optional prop it should be `multiple?: boolean;`, otherwise the default value below is never used.